docs/io.cozy.contacts.md

```diff
@@ -20,7 +20,8 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
 - `note?`: {string}
 - `email?`: {array} An array of email addresses objects with the following attributes:
   - `address`: {string} Email address
-  - `type?`: {string} A user-provided localized type of email address (example: `"Work"`, `"Home"`, `"Other"`)
+  - `type?`: {string} A user-provided localized type
```

Comment:
Why removing the examples?

Reply:
Work et Home ne sont pas très pertinent maintenant qu'ils sont portés par le label qui gère le home/work justement. On peut mettre ce qu'on veut dans le type, et je n'ai pas trouvé de bonnes idées pour illustrer. Je me suis dit que c'était suffisant en l'état.

On peut mettre sinon "Perso / Pro" ou "Personnal / Profressional" ? Comme tu veux :)

Reply:
"Personnal / Profressional" me paraît bien. Ca me parait utile d'avoir un  exemple, surtout que `type` et `label` semblent sémantiquement inversé, il me paraît plus naturel d'avoir `type: work` et `label:  "Pro"` que l'inverse, par exemple